src/jit/compiler.cpp

```diff
@@ -4815,8 +4815,8 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
     m_pLinearScan = getLinearScanAllocator(this);
 
     /* Lower */
-    Lowering lower(this, m_pLinearScan); // PHASE_LOWERING
-    lower.Run();
+    m_pLowering = new (this, CMK_LSRA) Lowering(this, m_pLinearScan); // PHASE_LOWERING
```

Comment:
Yes, it's unfortunate, but any phase that runs after `Lowering` must re-run `Lowering` on any newly introduced IR. This was previously not an issue, due to the small amount of code that's introduced post-`Lowering`, but liveness introduces new local stores, which need to be transformed by the type widening. It will also impact some code in `fgUpdateFlowGraph` and its descendants once we are doing containment analysis in `Lowering`.